dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.39/V2_39_14__Add_referral_column_to_relationshiptype.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE relationshiptype ADD COLUMN IF NOT EXISTS referral BOOLEAN DEFAULT false;
```

Comment:
I think you can have the SET DEFAULT (3rd script) inlined with your first script. It will still be idempotent and will not fail if the column already exists and even if it has null values.